tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -22,6 +22,11 @@
 import pyotp
 import pytest
 from airgun.session import Session
+from automation_tools import enable_repos
+from automation_tools import subscribe
+from automation_tools import unsubscribe
+from fabric.api import env
+from fabric.api import execute
```

Comment:
Please don't include fabric and automation-tools into robottelo. 
(This is not upgrades)

You are lacking `Fabric3` in `requirements.txt` anyway